src/Symfony/Bundle/FrameworkBundle/Tests/Command/RouterMatchCommandTest.php

```diff
@@ -78,11 +78,7 @@ private function getKernel()
             ->expects($this->atLeastOnce())
             ->method('has')
             ->will($this->returnCallback(function ($id) {
-                if ('console.command_loader' === $id) {
```

Comment:
this applies to 3.4 also - maybe others too? they should be submitted against 3.4

Reply:
I'll re-check the PR changes against 3.4, thanks for reminding =)